packages/frontend/src/styles/custom-components.css

```diff
@@ -1,23 +1,28 @@
+.praise-defaults {
+  @apply w-full px-0 py-5 md:px-5;
+}
+
 .praise-page {
-  @apply max-w-5xl mx-auto
+  @apply praise-defaults md:w-[750px];
 }
 
-.praise-box {
-  @apply p-5 mb-5 break-words border rounded-lg shadow-sm bg-gray-50;
+.praise-page-wide {
+  @apply praise-defaults md:w-[750px] xl:w-[1000px];
 }
 
-.praise-button {
-  @apply px-4 py-2 font-bold text-white bg-gray-800 rounded hover:bg-gray-700;
+.praise-box-defaults {
+  @apply border shadow-md rounded-xl bg-warm-gray-50 dark:bg-slate-600;
 }
 
-.praise-button-disabled {
-  @apply px-4 py-2 font-bold text-gray-500 bg-gray-700 rounded cursor-default;
+.praise-box {
+  @apply praise-box-defaults w-full md:w-[710px] p-5;
 }
 
-.praise-button-round {
-  @apply flex items-center justify-center rounded-full hover:bg-gray-300 w-7 h-7 dark:text-white dark:hover:bg-slate-800;
+.praise-box-wide {
+  @apply praise-box-defaults w-full md:w-[710px] xl:w-[960px] p-5;
 }
```

Comment:
for all these `.praise-` classes: 

- can we use tailwinds sizing rather than hardcoding pixels?
- what do you think about renaming them to not use the prefix "praise-"? Instead perhaps "content-", "styled-", "app-", or "common-"? I feel like it gets a little confusing when we also have Praise component

Reply:
> for all these `.praise-` classes:
> 
> * can we use tailwinds sizing rather than hardcoding pixels?

See previous comment regarding sizes.

> * what do you think about renaming them to not use the prefix "praise-"? Instead perhaps "content-", "styled-", "app-", or "common-"? I feel like it gets a little confusing when we also have Praise component

Yes, let's come up wit a better naming structure. Out of scope for this PR though, let's make a new issue.